### ISSUE TITLE
task nativeRun or nativeOptimizedRun is not working on Windows OS

### ISSUE THREAD

**Issue by nassimus26** [micronaut-projects/micronaut-gradle-plugin#380]
### Expected Behavior

On windows 10 
I create a Micronaut project with 

`mn create-app demo --lang kotlin`

I have a linux sub system (WSL) I run on it : 
`./Gradlew nativeRun  --stacktrace`

The server run well with the message :
`15:26:07.863 [main] INFO  io.micronaut.runtime.Micronaut - Startup completed in 871ms. Server Running: http://localhost:8080`

I should get the same when I run the same command on windows

### Actual Behaviour

When I run the same command on windows I got : 

```
* Exception is:
org.gradle.internal.execution.WorkValidationException: A problem was found with the configuration of task ':nativeRun' (type 'NativeRunTask').
  - In plugin 'org.graalvm.buildtools.gradle.NativeImagePlugin$Inject' type 'org.graalvm.buildtools.gradle.tasks.NativeRunTask' property 'image' specifies file 'C:\work\micronaut-kotlin\build\native\nativeCompile\mk-service' which do
esn't exist.

    Reason: An input file was expected to be present but it doesn't exist
```
OK the nativeCompile succeed but not the nativeRun task

but the real problem is that when I run the generated EXE inside the `\build\native\nativeCompile` floder    

I got this message :

`15:46:45.049 [main] INFO  io.micronaut.runtime.Micronaut - No embedded container found. Running as CLI application`

### Steps To Reproduce

On windows 10
1- `mn create-app demo --lang kotlin`
2- gradlew :nativeCompile --stacktrace

### Environment Information

- Windows 10
- GraalVM CE 22.0.0.2 (build 17.0.2+8-jvmci-22.0-b05)

### Version

3.0.0

**Comment by melix** [comment]
Would you mind checking with plugin version `3.2.1`? Thanks!

**Comment by nassimus26** [comment]
yes even with  :
```
    id("io.micronaut.application") version "3.2.1"
    id("io.micronaut.aot") version "3.2.1"
```
it gives the same result for the :`nativeCompile `or the :`nativeOptimizedCompile`

**Comment by nassimus26** [comment]
@melix the first part of the problem is related to the `org.graalvm.buildtools.native` plugin

https://github.com/graalvm/native-build-tools/issues/198

**Comment by melix** [comment]
Fixed via #387
